### DISCUSSION TITLE
Does this library support Testing Library that is now natively supported in Playwright?

### DISCUSSION THREAD

**Discussion by costa-collibra** [lariat-js/playwright#44] [Q&A] [answered]
Looking at the docs Im not sure how this library works with Testing Library.

Please can you confirm?

Thanks

**Comment by mskelton** [comment, marked as answer]
Lariat does support testing library style methods as supported by Playwright.  It doesn't work with testing library, that's not the idea, it has methods that resemble the testing library queries.

https://playwright.dev/docs/locators#quick-guide

**Reply by costa-collibra** [reply]
Thanks sounds like it could be useful for us
